scale/node/test/test_views.py

```diff
@@ -183,16 +221,135 @@ def test_update_active(self):
             'is_active': False,
         }
 
-        url = rest_util.get_url('/nodes/%d/' % self.node2.id)
+        url = '/v5/nodes/%d/' % self.node2.id
         response = self.client.patch(url, json.dumps(json_data), 'application/json')
         self.assertEqual(response.status_code, status.HTTP_200_OK, response.content)
 
         result = json.loads(response.content)
         self.assertEqual(result['is_active'], False)
         self.assertIn('deprecated', result)
 
+class TestNodeDetailsViewV6(TransactionTestCase):
+
+    def setUp(self):
+        django.setup()
 
-class TestNodesStatusView(TransactionTestCase):
+        self.node1 = node_test_utils.create_node()
+        self.node2 = node_test_utils.create_node()
+        self.node3 = node_test_utils.create_node()
+
+        Scheduler.objects.create(id=1, master_hostname='localhost', master_port=5050)
+
+    @patch('mesos_api.api.get_slave')
```

Comment:
Remove all of these mock matches for v6. The v6 API should not rely on any Mesos API calls (I don't think v5 does either...)